content/_index.md

```diff
@@ -1,23 +1,80 @@
+2. Embed an id for that artifact tree into the artifact being built.
+
+in a language-heterogenous environment with zero developer effort, involvement, or awareness.
+
+- GitBOM is not an SBOM, though it is designed to complement existing SBOM efforts such as [SPDX](https://spdx.dev/), [CycloneDX](https://cyclonedx.org/), or [SWID](https://nvd.nist.gov/products/swid).
+- GitBOM is not a version control system. It is designed to co-exist with them, but not require them.
+
+GitBOM applies the [Unix Philosophy](https://en.wikipedia.org/wiki/Unix_philosophy). It does one thing well: constructing [artifact trees](/glossary/artifact_tree).
+
+<!-- GitBOM borrows the use of [merkle tree](/resources/glossay/#merkle-tree) and the [gitref](/resources/glossay/#merkle-tree) from [git](/resources/glossay/#mgit) to construct a [verifiable bare artifact tree](/resources/glossay/#verifiable-bare-artifact-tree) in a manner suitable for [primary build tools](/resources/glossay/#primary-build-tools) to compute the GitBOM for an artifact and embed a unqiue, immutable, verifiable GitBOM id in the artifact being built. 
+in language-heterogenuous environments with zero-end-user-effort. -->
 
-* Lorem Ipsum1
-* Lorem Ipsum2
 {{< /home/section >}}
+<!-- 
+{{< home/section title="GitBOM is not:" iconclass="fas fa-2x fa-power-off">}}
+1. Not a system for build reproducability, but it does provide information that is useful for that.
+2. Not a version control system, though it is designed to co-exist with them.
+3. Not an SBOM, though it is designed to complement them.
+4. Not a globally unique software identifier (SWID).
+5. Not reliant on any particular packaging or distribution mechanism, either for artifacts or for artifact identity graphs).
+{{< /home/section >}} -->
+
+
+{{< home/section title="Why?" >}}
+A complete artifact tree with absolute precision all the way down to the individual source code files enables:
+
+- Detecting potential vulnerabilities in executables/containers.
+- Identifying Open Source License obligations
+- Identifying commercial license obligations
+- More reliable attestation
+- Post exploit forensics
 
-{{< home/section title="Use Cases" >}}
-GitBOM is...
+<!-- There are many many many use cases that could use GitBOMs.  An incomplete list would include:
 
-* Lorem Ipsum1
-* Lorem Ipsum2
+* Detecting potential vulnerabilities in executables/containers.
+* Identifying Open Source License obligations
+* Identifying commercial license obligations
+* More reliable attestation
+* Post exploit forensics -->
 {{< /home/section >}}
 
-{{< home/section title="How it works">}}
-GitBOM is...
+{{< home/section title="How?">}}
+GitBOM observes:
+
+1. Every artifact is a [blob](/resources/glossay/#merkle-tree)
+2. Every blob has a [gitref](/resources/glossay/#merkle-tree)
+3. Therefore the gitref may be used as an artifact id for an artifact
+4. Most source code artifacts are stored already with their gitref as their id
```

Comment:
```suggestion
4. Most source code artifacts are stored with their gitref as their on-disk identifier
```

Reply:
I don't quite follow the 'on-disk' part of this.  I can totally get behind expanding 'id' to 'identifier' ... but curious what the thought process was with 'on-disk'

Reply:
I was thinking of different ways that executable objects are identified on-disk/at-rest versus when in use. On second look, I think my own suggestion adds more confusion here than clarity.